launcher/control.py

```diff
@@ -187,18 +195,24 @@ def command(self, command):
     @Slot(QtCore.QModelIndex)
     def push(self, index):
         name = model.data(index, "name")
-        self.breadcrumbs.append(name)
 
-        level = len(self.breadcrumbs)
-        handler = {
-            1: self.on_project_changed,
-            2: self.on_silo_changed,
-            3: self.on_asset_changed,
-            4: self.on_task_changed
-        }[level]
+        frame = self.current_frame()
+
+        # If nothing is set as current frame we are at selecting projects
+        handler = self.on_project_changed
+        if frame:
+            handler = {
+                "project": self.on_asset_changed,
+                "silo": self.on_asset_changed,
+                "asset": self.on_asset_changed
+            }[frame["type"]]
```

Comment:
It look like `frame` in first launch did not have key `"type"`, so the above if-statement should change into
```python
if frame and frame.get("type"):
    ...
````